Multisig/Data/Services/Utils/GSError.swift

```diff
@@ -116,8 +116,9 @@ enum GSError {
             case 50:
                 return UnprocessableEntity(reason: "Safe info is not found.", code: 42250)
             default:
-                LogService.shared.error("Unrecognised error with code: \(error.code); message: \(error.message)",
-                                        error: unexpectedError)
+                LogService.shared.error(
+                    "Unrecognised error with code: \(error.code); message: \(String(describing: error.message))",
```

Comment:
if the message is optional, will the text have the "message: Optional<nil>" or will it be empty?

Reply:
good catch

Reply:
fixed